core/pom.xml

```diff
@@ -114,6 +114,7 @@
     <module>jms-artemis</module>
     <module>jpa-support</module>
     <module>jpa</module>
+    <module>hibernate-search</module>
```

Comment:
Also breaks alphabetical order.

Reply:
Will correct. I was not aware there was a convention

Reply:
More a convenience than convention